backends/bmv2/jsonconverter.cpp

```diff
@@ -803,13 +805,34 @@ JsonConverter::JsonConverter(const CompilerOptions& options) :
         headerParameter(nullptr), userMetadataParameter(nullptr), stdMetadataParameter(nullptr)
 {}
 
+void addSrcInfoData(Util::JsonObject* result,
```

Comment:
Some comments claim that SourceFileLine contains column numbers, but the implementation does not.  Should that be added?

Also, are you suggesting that _every_ node have and print a SourceFileLine object?  I'm not sure that is what we want for nested expression trees, for example (((x+5) ^ (y-7)) - 1) should probably have source info once for the root node, not for every sub-node.

Also there are nodes synthesized during compilation that either shouldn't have, or might in general be difficult to create for them, source location information.  Are you suggesting that for such nodes the toJson() method should be a no-op?